src/engine/strat_engine/backstore/metadata/bda.rs

```diff
@@ -31,6 +31,7 @@ impl BDA {
     /// Initialize a blockdev with a Stratis BDA.
     pub fn initialize<F>(
         f: &mut F,
+        index: usize,
```

Comment:
Try renaming ```index``` to ```offset```. You'll also want to document the meaning of this parameter. There is no particular format that we enforce with documentation in our Rust projects, unlike w/ our Python projects, where we use the sphinx markup.